lib/matplotlib/figure.py

```diff
@@ -2684,6 +2888,60 @@ def get_size_inches(self):
         """
         return np.array(self.bbox_inches.p1)
 
+    def set_figsize(self, w, h=None):
```

Comment:
This should probably just use `cbook._define_aliases` (see other uses in the library).

Reply:
Thanks @anntzer. I made the requested change in the newest commit. Also updated the tests to match pep8.